### PR TITLE
Removing Muon POG DQM sequence from Express pipeline - backport to 14_0_X

### PR DESCRIPTION
**Pull request by rseidita**
#### PR description:

Removing Muon POG DQM sequence from Express pipeline to reduce memory load at T0.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Backport of #46293

### CODE REVIEW

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
please abort

**Comment by fabiocos** [comment]
please test workflow 1001.3,1001.4

**Comment by antoniovagnerini** [comment]
backport of https://github.com/cms-sw/cmssw/pull/46293

**Comment by antoniovagnerini** [comment]
+1

**Comment by antoniovilela** [comment]
+1

File: DQMOffline/Configuration/python/DQMOffline_cff.py [hunk 1]
```diff
@@ -158,7 +158,7 @@
 
 
 DQMOfflinePrePOGExpress = cms.Sequence( DQMOfflineTracking *
-                                 DQMOfflineMUO *
```
Comment (mmusich) [inline DQMOffline/Configuration/python/DQMOffline_cff.py:161]: https://github.com/cms-sw/cmssw/blob/6da11027b7ec7b8220f266521178030faccc3d71/DQMOffline/Configuration/python/DQMOffline_cff.py#L132

and

https://github.com/cms-sw/cmssw/blob/6da11027b7ec7b8220f266521178030faccc3d71/DQMOffline/Muon/python/muonMonitors_cff.py#L25-L30

I think this removed more than it was meant to (see https://cms-talk.web.cern.ch/t/missing-folder-in-streamexpress-run2024i-express-v2-dqmio/52718/1). @fabiocos @ptcox

Comment (rseidita) [reply DQMOffline/Configuration/python/DQMOffline_cff.py:161]: Indeed, I missed this, will fix it ASAP. Apologies.

**Pull request merged**